src/platform/endpoint/node/responsesApi.ts

```diff
@@ -248,7 +248,7 @@ class OpenAIResponsesProcessor {
 							// CAPI models don't stream the reasoning summary for some reason, byok do, so don't duplicate it
 							text: this.hasReceivedReasoningSummary ?
 								undefined :
-								chunk.item.summary.map(s => s.text).join('\n\n'),
+								chunk.item.summary.map(s => s.text),
```

Comment:
This should also be fixed in the `summary_text.delta` handling below.

Reply:
`chunk.delta` is already a string here though, and we get it back as a `string` not a `string[]`

Reply:
Discussed offline. This will be updated as a follow up PR.

essentially, the updated point here is handling a CAPI problem where we are not getting streaming reasoning summary text. But, when using BYOK/BYOE you actually get `summary_text.delta` with `summary_index`. That could be overlapping, so that has to be handled to avoid mixing summaries between different indexes.